spark/dl/src/main/scala/com/intel/analytics/bigdl/optim/DistriOptimizerV2.scala

```diff
@@ -202,14 +206,13 @@ object DistriOptimizerV2 extends AbstractOptimizer {
         recordsNum += results.records
 
         Iterator.single(results.successed)
-      }
-
-    val successModels = trainingTrace.traceIteration(training.reduce(_ + _))
-
+      }.reduce(_ + _)
+    
     parameterSync(lossSum.value, successModels, cacheOfMaster, models, context)
-
+    val endTime = System.nanoTime() - startTime
+    
     driverStatesUpdate(cacheOfMaster, recordsNum.value,
-      context, trainingTrace, metrics)
+      context, trainingTrace, endTime, metrics)
```

Comment:
why change the time counting?

Reply:
each iterationTakes should be reduce() + parameterSync()
previous `trainingTrace.traceIteration()` can only hold reduce() time.

Reply:
Can we add a new metric for parameter sync?

Reply:
Done. Put `reduce() + parameterSync()` in `trainingTrace.traceIteration()` to count full time iteration takes.